src/login.controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Body, Controller, HttpCode, Post, HttpStatus } from '@nestjs/common';
+import { LoginService } from './login.service';
+
+@Controller('login')
+export class LoginController {
+  constructor(private readonly loginService: LoginService) {}
+
+  @Post()
+  @HttpCode(HttpStatus.OK)
+  login(
+    @Body('username') username: string,
+    @Body('password') password: string,
+  ) {
+    return this.loginService.login(username, password);
+  }
```

Comment:
Since you need to add `openapi` here, you need to move both of this params into a DTO `LoginDto`.
```suggestion
  login(@Body() loginDTO: LoginDTO) {
    return this.loginService.login(loginDTO);
  }
```

Reply:
Resolved on 405d4653a891f3960b46d94eb9d29f2ce3d075de.